### ISSUE TITLE
@PropertyName is ignored in Shadows creation

### ISSUE THREAD

**Issue by bartoszwalacik** [javers/javers#574]
it causes unexpected nulls in Shadows, reproduction scenario:

```
class CaseShadowWithPropertyName extends Specification {

    class Entity {
        @Id int id
        @PropertyName("otherField") String someField
    }

    def "should use @PropertyName when creating Shadows"(){
      given:
      def javers = JaversBuilder.javers().build()
      def e = new Entity(id:1, someField: "s")
      javers.commit("author", e)

      when:
      def shadow = javers.findShadows(QueryBuilder.byInstance(e).build()).get(0).get()

      then:
      shadow instanceof Entity
      shadow.id == 1
      shadow.someField == "s"
    }
}
```

**Comment by pszymczyk** [comment]
Is last line in then section correct? Did you mean `shadow.someField == "s"` or `shadow. otherField == "s"` ?

**Comment by bartoszwalacik** [comment]
yes, it's correct.
shadow.someField isn't populated

**Comment by pszymczyk** [comment]
The simplest workaround is to annotate field with Gson annotation `@SerializedName("otherField")` but it's not final solution for sure.

**Comment by bartoszwalacik** [comment]
fix released in 3.5.2
